bin/config-optimist.js

```diff
@@ -38,6 +38,7 @@ module.exports = function(optimist) {
 		.describe("optimize-max-chunks", "Try to keep the chunk count below a limit")
```

Comment:
BTW, it seems like `config-optimist.js` is not used anywhere. Should we just remove it?

Reply:
It will be removed when migrating to webpack-cli.